data-analysis/introduction/data-analysis-using-spreadsheets/data-cleaning.md

```diff
@@ -0,0 +1,82 @@
+---
+author: emmab
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+links:
+
+	- '[Organize Your Spreadsheets](https://towardsdatascience.com/organize-your-spreadsheets-for-data-analysis-and-visualization-b1985a31523a){website}'
+
+
+---
+# Data Cleaning
+
+---
+## Content
+
+To make spreadsheets a useful tool for data analysis, you first need to **organize the data inside** the sheets.
+
+You should keep your raw source data in one tab, untouched. Then in one or several other tabs, have the views and analysis of the original data.
+
+If your raw data has already been tampered with and you can't download it again, follow these steps to revert it back to a raw state:
+
+1. **Unpretty your spreadsheets**
+
+- Remove merged cells as these can’t be used in pivot tables and are not well supported by external tools
+- Remove multi-line headers
+- Remove titles, formatting, file version, date and authors (use the file history and versioning for that)
+
+2. **Untidy your data**
+
+- Each row must represent one unaggregated measure
+- Each cell must contain only one dimension or measure
+- Each column must have a unique name
+
+# More Tips 
```

Comment:
- h1 (`#`) is reserved for insight headline
- h2 (`##`) is reserved for section titles (e.g. `## Practice`)

you can use h3 (`###`) inside the content section (`## Content`) meaning this should be
```
### More Tips
```